src/Container/Docker/Manifest.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Container.Docker.Manifest (
+  ManifestJson (..),
+  ManifestJsonImageEntry (..),
+  decodeManifestJson,
+  getLayerPaths,
+  manifestFilename,
+) where
+
+import Control.DeepSeq (NFData)
+import Data.Aeson (
+  FromJSON,
+  ToJSON,
+  defaultOptions,
+  eitherDecode,
+  genericToEncoding,
+  parseJSON,
+  toEncoding,
+  withObject,
+  (.!=),
+  (.:),
+ )
+import Data.ByteString.Lazy qualified as ByteStringLazy
+import Data.List.NonEmpty qualified as NonEmpty
+import Data.Text (Text)
+import GHC.Generics (Generic)
+
+manifestFilename :: Text
+manifestFilename = "manifest.json"
```

Comment:
Is the file guaranteed to be named `manifest.json`? Even if it can be modified I don't think we should support it until necessary, I'm just curious.

Reply:
yes - this is part of Spec for tarball exported docker image.